pages/index.tsx

```diff
@@ -1,5 +1,11 @@
 import Head from 'next/head'
 
+import Logo from 'components/Logo/Logo'
+
+export const config = {
+  unstable_runtimeJS: false,
```

Comment:
this is an [experimental flag](https://github.com/vercel/next.js/pull/11949) I discovered while looking into how to minimize Next's export bundle. We can use this to prevent NextJS loading scripts to "rehydrate" the client for pages with truly no JS. There's more discussion about this feature [here](https://github.com/vercel/next.js/issues/5054). We will probably end up removing this once we start adding actual client-side JS but for now it keeps the exported site as small as possible!